README.md

```diff
@@ -182,8 +182,17 @@ entirely at a future date.
 
 #### Alternative Setup: Docker
 
-It is also possible to run load tests from within a Docker container, eliminating the need to set up a valid python
-environment. This requires Docker and `docker-compose` to be installed on your machine. Get them here:
+It might be possible to run load tests from within a docker container, though given recent updates this seems to not be
```

Comment:
Nit: maybe consider dropping "recent" since some time might pass before this is looked at again. Also, it might be worth linking to the PR's that introduced the breaking changes here